lua/astrocommunity/editing-support/true-zen-nvim/init.lua

```diff
@@ -1,5 +1,6 @@
 return {
   "Pocco81/true-zen.nvim",
+  event = "VeryLazy",
```

Comment:
```suggestion
```



This should not be lazy loaded on event. It is lazy loaded on module and keybinding